Meadowlark-js/tests/http/local.deleteissue.http

```diff
@@ -115,7 +115,16 @@ reference-validation: true
         }
     ]
 }
-###
+### This delete will fail because the school is referenced by the location and the ed-org is referenced 
+# by the course
 @schoolDocumentlocation = {{school124.response.headers.location}}
 DELETE http://localhost:3000{{schoolDocumentlocation}}
+{{auth_header}}
+
```

Comment:
these tests got us to the point of showing that once you references a school, you could not delete it, but left it there. Added a delete for location and course objects allowing the school to be deleted